21-Hello-World/05-Application-Structure/src/02-MTL/02-Implementing-a-Monad-Transformer/03-A-Magical-Monad.md

```diff
@@ -192,53 +208,7 @@ instance ms :: (Monad m) => Monad (StateT state monad) where
   state f = StateT (\s -> pure $ f s)
 ```
 
-## FAABM Using Bind
 
-Notice, however, that the above `let ... in` syntax is really just a verbose way of doing `bind`/`>>=`. If we were to rewrite our instances using `bind`, they now look like this:
-
-```purescript
-instance map :: (Monad monad) => Functor (StateT state monad) where
-  map :: forall a b
-       . (a -> b)
-      -> StateT state monad a
-      -> StateT state monad b
-  map f (StateT g) = StateT (\s1 ->
-      (g s1) >>= (\(Tuple value1 s2) ->
-        pure $ Tuple (function value1) s2
-      )
-    )
-
-instance apply :: (Monad monad) => Apply (StateT state monad) where
-  apply :: forall a b
-        -- (state -> Tuple (a -> b) state)
-         . StateT state monad (a -> b)
-        -> StateT state monad a
-        -> StateT state monad b
-  apply (StateT f) (StateT g) = StateT (\s1 ->
-      (g s1) >>= (\(Tuple value1 s2) ->
-        (f s2) >>= (\(Tuple function s3) ->
-          pure $ Tuple (function value1) s3
-        )
-      )
-    )
-
-instance apctv :: (Monad monad) => Applicative (StateT state monad) where
-  pure :: forall a. a -> StateT state monad a
-  pure a = StateT (\s -> pure $ Tuple a s)
-
-instance bind :: (Monad monad) => Bind (StateT state monad) where
-  bind :: forall a b
-        . StateT state monad a
-       -> (a -> StateT state monad b)
-       -> StateT state monad b
-  bind (StateT g) f = StateT (\s1 ->
-      (g s1) >>= (\(Tuple value1 s2) ->
-        let (StateT h) = f value1 in h s2
-      )
-    )
-
-instance monad :: (Monad m) => Monad (StateT state monad)
-```
```

Comment:
Why was this entire section deleted?

Reply:
Sorry about this. In last comment, I demonstrated how to use bind/>>= to implement functor and apply type class separately in related section. so I supposed this section is not required. 
I already added it back.